wizard/Dockerfile

```
@@ -0,0 +1,36 @@
+# wizard/Dockerfile
+# Build context: repo root (needs wizard/ AND sample_kb/)
+FROM python:3.11-slim
+
+WORKDIR /app
+
+# Install Docker CLI + Compose plugin (needed to launch app containers via socket)
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)